src/components/review/ReviewsList.js

```diff
@@ -80,13 +80,21 @@ const AlbumReviews = ({ albumId }) => {
         />
       )}
       {user && userHasReviewed && (
-          <UpdateReview reviewId={
-            reviews.find((review) => review.user === user?.user?._id)?._id} refreshReviews={refreshReviews} />
-        )}
-       {user && userHasReviewed && (
-          <DeleteReview reviewId={
-            reviews.find((review) => review.user === user?.user?._id)?._id} refreshReviews={refreshReviews} />
-        )}
+        <UpdateReview
+          reviewId={
+            reviews.find((review) => review.user === user?.user?._id)?._id
+          }
+          refreshReviews={refreshReviews}
+        />
+      )}
```

Comment:
This works well 👍🏾👍🏾

One improvement we could try is instead making it so we can click on our review to edit it; and only then show this `UpdateReview` component. 

https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/271b8dfe-a4bd-4d3a-af87-8789da200c43

Reply:
@akosasante  I think I did it! check it out (as well as a delete btn)